.github/workflows/test_upload_build_to_test_flight.yml

```diff
@@ -14,7 +14,7 @@ on:
 jobs:    
   build:
     name: Build
-    runs-on: macOS-latest
+    runs-on: macOS-12
```

Comment:
Just curious why do we need to make this change to run specifically on `macOS-12` only? Isn't the fix for supporting the latest environment? 🤔

Reply:
macOS-lastest is [macOS-11](https://github.com/actions/runner-images/blob/main/images/macos/macos-11-Readme.md) and does not have Xcode 14. We should change to macOS-lastest when Actions update the default [macOS version ](https://github.com/actions/runner-images/issues/6384).

Reply:
@blyscuit that is a great insight! Then I think you can also include it as a part of the `Insight` section in the PR description for later action also 👍 Rest LGTM 🚀